src/components/ExpenseForm.js

```diff
@@ -119,13 +176,22 @@ const Form = styled.form`
     -webkit-appearance: none;
     -moz-appearance: textfield;
   }
-  & > button {
+  & button {
```

Comment:
you can omit the & selector
```suggestion
  button {
```